lib/admin/user.rb

```diff
@@ -20,7 +12,7 @@ def self.authenticate(params = {})
       return nil if username != ENV['USERNAME']
 
       password_hash = Password.new(ENV['PASSWORD_HASH'])
-      User.new(username) if password_hash == params[:password] # The password param gets hashed for us by the == method.
```

Comment:
Why is the comment gone? Was it incorrect? If it wasn't, that was an excellently useful comment. If it was, where is the password param hashed?